dato/js/recview.js

```diff
@@ -0,0 +1,158 @@
+$(document).ready( function(event){
```

Comment:
I believe jQuery is currently a dependency of some sections of the userguide, rather than GitBook, e.g. 
https://github.com/dato-code/userguide/blob/74b334f53e556c044283e1a7c84868360ff5e598/supervised-learning/scripts/picture-canvsas.html#L7